docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,518 @@
+#### Import packagages and modules
+
+First we load standard packages,
+
+```julia
+using Distributions
+using StatsBase
+using LinearAlgebra
+using StatsPlots
+using Plots
+using Plots.PlotMeasures
+using Random
+using JLD2
+```
+
+the Cloudy modules,
+
+```julia
+using Cloudy
+using Cloudy.ParticleDistributions
+using Cloudy.KernelTensors
+
+# Import the module that runs Cloudy
+include(joinpath(@__DIR__, "DynamicalModel.jl"))
+```
+
+and finally the EKP packages.
+
+```julia
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.Observations
+using EnsembleKalmanProcesses.ParameterDistributions
+using EnsembleKalmanProcesses.DataContainers
+```
+
+The module `DynamicalModel.jl` is the forward solver; it provides a function
+that runs Cloudy with a given instance of the parameter vector we want to learn.
+
+```julia
+include("DynamicalModel.jl")
+```
+
+#### Define the true parameters
+
+We define the true parameters---they are known here because this is a known
+truth example. Knowing the true parameters will allow us to assess how well
+Calibrate-Emulate-Sample has managed to solve the inverse problem.
+
+We will assume that the true particle mass distribution is a Gamma distribution,
+which at time ``t = 0`` has parameters ``\phi_0 = [N_{0, 0}, k_0, \theta_0]``. We will then try to learn these parameters from observations ``y = [M_0(t_{end}), M_1(t_{end}), M_2(t_{end})]`` of the zeroth-, first-, and second-order moments
+of the distribution at time ``t_{end} > 0`` (where `t_end = 1.0` in this
+example). The true parameters ``\phi_{0, \text{true}}`` are defined as follows:
+
+```julia
+param_names = ["N0", "θ", "k"]
+n_params = length(param_names)
+N0_true = 300.0  # number of particles (scaling factor for Gamma distribution)
+θ_true = 1.5597  # scale parameter of Gamma distribution
+k_true = 0.0817  # shape parameter of Gamma distribution
+ϕ_true = [N0_true, θ_true, k_true] # true parameters in constrained space
+dist_true = ParticleDistributions.GammaPrimitiveParticleDistribution(ϕ_true...)
+```
+
+#### Define priors for the parameters
+
+As we are working with Bayesian methods, we treat the parameters we want to
+learn as random variables whose prior distributions we specify here. The prior
+distributions will behave like an "initial guess" for the likely region of
+parameter space where we expect the solution to be located.
+We use `constrained_gaussian` to add the desired scale and bounds to the prior distribution, in particular we place lower bounds to preserve positivity (and numerical stability). 
+
+```julia
+# We choose to use normal distributions to represent the prior distributions of
+# the parameters in the transformed (unconstrained) space.
+prior_N0 = constrained_gaussian(param_names[1], 400, 300, 0.4 * N0_true, Inf)
+prior_θ = constrained_gaussian(param_names[2], 1.0, 5.0, 1e-1, Inf)
+prior_k = constrained_gaussian(param_names[3], 0.2, 1.0, 1e-4, Inf)
+priors = combine_distributions([prior_N0, prior_θ, prior_k])
+```
```

Comment:
you could put in the command `plot(prior)` and the figure (given in physical space by default), here.